### ISSUE TITLE
Too many request issue in AzoraMoon

### ISSUE THREAD

**Issue by 1maul** [KotatsuApp/kotatsu-parsers#948]
### Source information

AzoraMoon

### Steps to reproduce

When opening or searching in AzoraMoon it gives me this error even though this source works in tachiyomi fork with no issues please fix this

### Kotatsu version

7.3

### Android version

9

### Other details

![Screenshot_20240727_162908_org koitharu kotatsu](https://github.com/user-attachments/assets/875ef1d2-327d-4167-b409-660493eee70c)
![Screenshot_٢٠٢٤٠٧٢٧_١٦٣٥١٣](https://github.com/user-attachments/assets/c8379df4-be64-452e-ad10-0f98ad590526)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by 1maul** [comment]
This issue was in the app long time ago and that's annoying

**Comment by dragonx943** [comment]
> This issue was in the app long time ago and that's annoying

Too much request error = Avoid having ur IP blocked by that website due to sending too many requests (spam)
Btw, u can turn on "Download slowdown" option for every sources to reduce that errors :p

![image](https://github.com/user-attachments/assets/557049a4-1653-46b3-87c5-9099151e5a0d)
